cps2pc

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+import argparse
+import cps
+from collections import OrderedDict
+import os
+import re
+import subprocess
+import sys
+
+_supported_compilers = ['gcc', 'clang']
+_default_prefix = '${pcfiledir}/../..'
+#------------------------------------------------------------------------------
+def get_link_features(link_features, platform):
+    _supported_kernels = ['cygwin', 'linux', 'darwin']
+    _supported_link_features = {x: {'threads': '-lpthread'} for x in _supported_kernels}
```

Comment:
While I am not trying to follow *all* of PEP-8, I do prefer to keep lines under 79 characters. This one and a number of others here are "too long".

(In fact, I think I am generally in favor of following `pycodestyle` *except* for E261, E265, E302, and E305... and I'm not convinced on E731. I'm willing to relax line length when the alternative is worse, especially for only one character over, but I don't believe that's the case for many of the over-length lines here.)

See also a34ffa1f5574.

Reply:
done.